src/braket/pennylane_plugin/ops.py

```diff
@@ -63,56 +55,6 @@
 from pennylane.ops.qubit import four_term_grad_recipe
 
 
-class CPhaseShift(Operation):

```

Comment:
Available in [PennyLane](https://github.com/PennyLaneAI/pennylane/blob/ca227436391adf10df1f946cc727a9abb36673eb/pennylane/ops/qubit.py#L898)